src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -748,7 +746,7 @@ class VMBootDiagnostics(ResourceGroupVCRTestBase):
     def __init__(self, test_method):
         super(VMBootDiagnostics, self).__init__(__file__, test_method, resource_group='cli_test_vm_diagnostics')
         self.vm_name = 'myvm'
-        self.storage_name = 'ae94nvfl1k1'
+        self.storage_name = 'clitestbootdiag20170227'
```

Comment:
Is this storage generated dynamically or statically? I concern the hard coded storage name.